config/300-broker.yaml

```diff
@@ -43,9 +43,9 @@ spec:
     - name: Reason
       type: string
       JSONPath: ".status.conditions[?(@.type==\"Ready\")].reason"
-    - name: Hostname
+    - name: URL
       type: string
-      JSONPath: .status.address.hostname
+      JSONPath: .status.address.url
```

Comment:
@nachocano with this change to `url`, I noticed a bit of inconsistency 

Right now we use same in:
* https://github.com/knative/eventing/blob/master/config/300-channel.yaml#L52 
* https://github.com/knative/eventing/blob/master/config/300-parallel.yaml#L45

but `hostname` in:
* https://github.com/knative/eventing/blob/master/config/channels/in-memory-channel/300-in-memory-channel.yaml#L51  
*  https://github.com/knative/eventing/blob/master/config/300-sequence.yaml#L48

Reply:
changed them to url. We should be using url instead... thanks for the catch...